Makefile

```diff
@@ -1,6 +1,6 @@
-CC = gcc
+CC = clang
```

Comment:
Any specific reason to use clang instead of gcc?

Reply:
Nope, this was a local change due to language server recognizing linting. I'll remove it.